src/test/java/com/iexec/common/docker/client/DockerClientInstanceTests.java

```diff
@@ -411,6 +413,14 @@ public void shouldPullImage() {
         dockerClientInstance.removeImage(ALPINE_LATEST);
     }
 
+    @Test
```

Comment:
There is not test for successful pulling of an image using the new method.

Reply:
True. This is indirectly tested by `shouldPullImage` but let's add another explicit test :)